lms/static/js/fixtures/instructor_dashboard/data_download.html

```diff
@@ -1,9 +1,252 @@
+                profile information for enrolled students directly on this page:")}</p>
+            %endif
+            <p hidden="hidden" class="selectionInfo reports profileInformation">${_("Click to generate a CSV file of \
+                all students enrolled in this course, along with profile information such as email address and \
+                username:")}</p>
+
+            <p hidden="hidden" class="selectionInfo reports learnerWhoCanEnroll">${_("Click to generate a CSV file \
+                that lists learners who can enroll in the course but have not yet done so.")}</p>
+
+            <p hidden="hidden" class="selectionInfo reports proctoredExamResults">${_("Click to generate a CSV file \
+                of all proctored exam results in this course.")}</p>
+
+            <p hidden="hidden" class="selectionInfo reports surveyResultReport">${_("Click to generate a CSV file of \
+                survey results for this course.")}</p>
+        </div>
+
+    </section>
+    %if settings.FEATURES.get('ENABLE_GRADE_DOWNLOADS'):
+    <section id="certificate" class="idash-section tab-data" aria-labelledby="header-cert">
+        <h6 class="mb-15" id="header-cert">
+            <strong>${_("NOTE")}: </strong>
+            Please select the report type and then click Download Report button
+        </h6>
+
+        <select class="report-type selector">
+            <option value="viewCertificates" data-csv="false"
+                    data-endpoint="${ section_data['get_issued_certificates_url'] }">View certificates
+            </option>
+            <option value="downloadCertificates" data-csv="true"
+                    data-endpoint="${ section_data['get_issued_certificates_url'] }">Download csv of
+                certificates
+            </option>
+        </select>
+        <button  class="mb-20 btn-brand download-report" type="button" value="download report">Download
+            report
+        </button>
+        <div>
+            <p>${_("Click to list certificates that are issued for this course:")}</p>
+        </div>
+    </section>
+    <section id="problem" class="idash-section tab-data" aria-labelledby="header-problem">
+        <h6 class="mb-20" id="header-problem">
+            ${_("Select a problem to generate a CSV \
+            file that lists all student answers to the problem. You also select a section or chapter to include \
+            results of all problems in that section or chapter.")}
+        </h6>
+        <div class="mb-15 problems">
+            ${static.renderReact(
+            component="ProblemBrowser",
+            id="react-block-listing",
+            props={
+            "courseId": course.id,
+            "excludeBlockTypes": ['html', 'video', 'discussion']
+            }
+            )}
+        </div>
+        <button data-endpoint="${ section_data['get_problem_responses_url'] }" id="download-problem-report"
```

Comment:
Can we keep the button style consistent? All existing buttons in the instructor dashboard are grey